### ISSUE TITLE
Can multiple regions be deployed asynchronously?

### ISSUE THREAD

**Issue by antonfelich** [PageUpPeopleOrg/microservice-bootstrap#40]

**Comment by jimmyp** [comment]
I thought we said we would just deploy to a single region? Did I mishear?

**Comment by humblelistener** [comment]
there are two things that are potentially meant here,
1. One ECR - deploying to multiple regions synchronously (one after another).
2. One ECR - deploying to multiple regions parallelly (reduces the deployment time)

I think this issue is about point 2 above. It will be good to get clarified

**Comment by jimmyp** [comment]
Yeah I think I'm saying I'd like to discuss avoiding both scenarios. Only ever deploying to one regsions as an opinion in the bootstrap. I think this should be discussed in #39 though and I jumped the gun starting this discussion here
